Documentation/Sprint 4/Sprint Plan.tex

```diff
@@ -14,28 +14,133 @@ \section*{Sprint Plan 4}
 
 
 \begin{table}[h]
-\begin{tabular}{|p{6cm}|p{5.5cm}|p{4cm}|p{3cm}|}
+\begin{tabular}{|p{4.5cm}|p{7cm}|p{4cm}|p{2.6cm}|}
 \textbf{User Story} & \textbf{Task} & \textbf{Task assigned to} & \textbf{Estimated Effort per Task} \\
-User Story 1 description & \begin{enumerate}
-\item task 1
-\item task 2
-\item task 3
+
+
+As a developer, I want to have a clear interface of the API calls between the frontend and the backend.
+
+&
+\begin{enumerate}
+\item Continue discussion on API contract.
+\item Update room selection API calls backend.
+\item Update room API calls frontend.
+\item Update chat API call backend.
+\item Update chat API call frontend.
+\end{enumerate}
+
+& 
+\begin{enumerate}
+\item Jaap Heijligers
+\item Gijs Weterings
+\item Tim van der Lippe
+\item Gijs Weterings
+\item Eva Anker
+\end{enumerate}
+
+& 
+\begin{enumerate}
+\item 10 hours
+\item 4 hours
+\item 4 hours
+\item 4 hours
+\item 4 hours
+\end{enumerate}
+\\
+
+As a user, I want to classify songs based on arousal and valence to improve the system.
+
+&
+\begin{enumerate}
+\item Implement frontend function thumbs up and down.
+\item Implement frontend function SAMs.
```

Comment:
We might need to add clarification in paranthesis what SAMs are. (I actually forgot what the abbrevation stands for ;p )